Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
```

Comment:
We need to do some adjustments to the project structure to have dedicated place for both of our Dockerfiles. Until then, please move this to cmd/operator